CoreVulkanDisplay/src/au/gov/asd/tac/constellation/visual/vulkan/CVKVisualProcessor.java

```diff
@@ -231,8 +229,9 @@ protected void initialise() {
 
     @Override
     protected void cleanup() {
-        // TODO_TT:
-        //canvas.removeGLEventListener(renderer);
+        cvkAxes.DestroyStaticResources();
```

Comment:
But why?  They are static, so destroying them with the instance makes no sense.

Reply:
As I said in my previous comment, we don't have a good place to do this yet. Will do in the work with the md5 hash checks.

Reply:
I don't think we should do any static cleanup.  Just remove it?  Either way, add a task to GitLab please to revisit the static cleanup (very low priority imo).

Reply:
Added to shader task